### ISSUE TITLE
Action chain failure should say on which action the chain failed.

### ISSUE THREAD

**Issue by lakshmi-kannan** [StackStorm/st2#1314]
See example: 

https://www.irccloud.com/pastebin/6AZmoQaS

Reported by: tobijb/jtimbrown on IRC

- st2 run commands should bubble up the failures in a more human readable way.  Status is failed below, but no error and the ordering of the output actually should have an entry for the execution that was never created and failed.  The error was that an action did not exist for the Mistral task that follows wait_for_environment.

```
vagrant@st2express:/opt/stackstorm/packs$ st2 run deployer.gittobi-trunk.deploy environment=master-jb.staging-tobi.com
..
id: 551981d59c993843801217df
action.ref: deployer.gittobi-trunk.deploy
status: failed
start_timestamp: 2015-03-30T17:03:17.596028Z

**Comment by Kami** [comment]
To clarify it, this is purely a CLI output formatting improvement - we should include failure message at the top to make it easy to see how a particular action chain task has failed without requiring user to run `st2 execution get <task execution id> to retrieve the details which include the error message,

Note - if there is a "top level error" (task references an invalid task, etc.), we already report this error as a top level error so it should already be displayed in the CLI output.

**Comment by Kami** [comment]
I've merged #1466 which should resolve this issue.
